common/changes/@microsoft.azure/openapi-validator-rulesets/tsp-DisableRules_2024-03-08-17-21.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You gotta bump the ruleset version in package.json and you can delete the change file that resulted from rush change. There is no need to run rush change. See my comment here:
https://github.com/Azure/azure-openapi-validator/pull/669/files#r1515048617

Reply:
Done.